### PR TITLE
How to Impersonate a Clever User on Localhost

### PR DESCRIPTION
**Pull request by Emily-ejag**
This guide walks you through the steps needed to impersonate a Clever user on your localhost environment for testing purposes.

### CODE REVIEW

**Comment by Emily-ejag** [comment]
CU-868a0ba3c

File: docs/workflows/test-clever-users.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+This guide walks you through the steps needed to impersonate a Clever user on your localhost for testing purposes.
+
+## Steps to Impersonate a Clever User
+
+1. **Start the Application in Development Mode**
+   - Run the following command on your terminal:
+     ```bash
+     npm run dev:live
+     ```
+   This will start the development server and prepare the app for Clever testing.
```
Comment (maximilianoertel) [inline docs/workflows/test-clever-users.md:12]: Could we elaborate why we need to start the application with `dev:live` and not the regular `dev`?

File: docs/workflows/test-clever-users.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+   - Log in to your Clever account.
+
+3. **Find and Select the User to Impersonate**
+   - In the Data Browser, search for the **district**, **school**, **sections (class)** and then select a **teacher**, **admin**, or **student** you want to impersonate.
+   - Click on the name of the user.
+   - Click on the blue button `Login as {User's Name}`.
+
+4. **Ensure Data Loading Completes**
+   - Once you click `Login as {User's Name}`, it will open the `roar.education` page wait until all data is fully loaded.
+   - You should be able to see the dashboard corresponding to the user you are impersonating.
```
Comment (maximilianoertel) [inline docs/workflows/test-clever-users.md:25]: Could we elaborate why we need to wait for data to fully load on the production environment before being able to access it locally?

**Review by maximilianoertel — COMMENTED** [review]
Looks good! Just two quick ones:

File: docs/workflows/test-clever-users.md (hunk 1 above)
Comment (Emily-ejag) [reply docs/workflows/test-clever-users.md:12]: yes!

File: docs/workflows/test-clever-users.md (hunk 2 above)
Comment (Emily-ejag) [reply docs/workflows/test-clever-users.md:27]: Yes!

**Review by maximilianoertel — APPROVED** [review]

**Review by lucasxsong — APPROVED** [review]

**Pull request merged**